src/app/containers/MediaPageMain/index.jsx

```diff
@@ -45,18 +31,47 @@ const MediaPageMain = props => {
       <main role="main">
         <Grid>
           <GridItemConstrainedMedium>
-            {blocks.map(renderBlock({ script, service }))}
-            <ul>
-              <li>
-                <strong>Service</strong>: {service}
-              </li>
-              <li>
-                <strong>Brand</strong>: {serviceId}
-              </li>
-              <li>
-                <strong>MediaId</strong>: {mediaId}
-              </li>
-            </ul>
+            {blocks.map(({ id, text, type, live, externalId }) => {
+              const blockType = live ? LIVE_RADIO_BLOCK : type;
+
+              switch (blockType) {
+                case HEADING_BLOCK:
+                  return (
+                    <Headline
+                      key={id}
+                      script={script}
+                      service={service}
+                      id="content"
+                    >
+                      {text}
+                    </Headline>
+                  );
+
+                case PARAGRAPH_BLOCK:
+                  return (
+                    <Paragraph key={id} script={script} service={service}>
+                      {text}
+                    </Paragraph>
+                  );
+
+                case LIVE_RADIO_BLOCK: {
+                  const MediaPlayer = {
+                    canonical: CanonicalMediaPlayer,
+                    amp: AmpMediaPlayer,
+                  }[platform];
+
+                  return (
+                    <MediaPlayer
+                      key={id}
+                      showPlaceholder={false}
+                      src={`/ws/av-embeds/media/${externalId}/${id}`}
```

Comment:
Isn't `id` a variable conflict here with the `id` being used in the key as well?

Reply:
EG: the payload has an `id` that is `liveradio` but each block is getting uuid added as `id` [here](https://github.com/bbc/simorgh/pull/3514/files#diff-a2570b611b8063fb122bb4d2b988442dR7)

Reply:
I think this might be related to why I'm seeing duplicate key console warnings 

![image](https://user-images.githubusercontent.com/7791726/64274300-ffe8a400-cf3a-11e9-9db7-acef213c3d20.png)

Reply:
I'm not seeing that error. Do you have the latest changes? I thought I fixed that with this commit https://github.com/bbc/simorgh/pull/3514/commits/badedd4af2325c10855bae20b78ec1b860492b46

Reply:
@pjlee11 thanks for finding those bugs. Should be addressed now.